server/src/test/java/org/elasticsearch/index/engine/InternalEngineTests.java

```diff
@@ -203,6 +203,11 @@
 
 public class InternalEngineTests extends EngineTestCase {
 
+    @Override
+    protected boolean enableWarningsCheck() {
```

Comment:
Warning check will only check for deprecated settings. These settings should be deprecated but remain in the tests until we fully moved on to soft-deletes.